### DISCUSSION TITLE
Getting sound from block

### DISCUSSION THREAD

**Discussion by malafk** [Minestom/Minestom#1301] [Help]
Hi I am trying to get the block sound as a string eg: "block.blocktype.break" so i can play it to a player

https://github.com/HorseNuggets/TerminatorPlus/blob/eca25d3355fc95fd0680e8cde26b5dc938971e8c/TerminatorPlus-Plugin/src/main/java/net/nuggetmc/tplus/bridge/InternalBridgeImpl.java#L25 

That code is basically what I am trying achieve but I have no clue how to using minestom

**Comment by emortaldev** [comment]
I don't believe there is an easy method for this, since that code uses NMS. It would be nice to have it in the block registry though.

However, if it's to fake a block being broken, you can use the effect packet
```java
new EffectPacket(
    2001, // Block break particles and break sound - see https://wiki.vg/Protocol#World_Event
    blockPosition,
    block.stateId(),
    false // Relative volume
)
```
